config.json.example

```diff
@@ -50,6 +50,13 @@
         "sell_profit_only": false,
         "ignore_roi_if_buy_signal": false
     },
+    "edge": {
+        "enabled": false,
+        "process_throttle_secs": 1800,
+        "maximum_winrate_to_consider": 0.80, 
+        "remove_pumps": true,
+        "minimum_delta_to_consider": 1
```

Comment:
A key metric is the % of users capital at risk in any 1 asset. 
Typically this should be 1 or 2% - this is not the capital in the trade which may be 100% of the users capital, but the capital that would be lost when stopped out. 

The combination of WR / R makes expectancy - the inclusion of position size based on limiting capital risk to the stop drives the statistical likelihood of dollar loss to the user.  It is risk-mgt

Without a maximum total_capital risk per asset the risk-mgt part is out of the users control. 
for example presume R and WR determine a 11% under spot is a good E - this could easily allow the bot to assign more than 20% of total user capital to a risk position. 

The idea is to need 50-100 bd trades to kill capital not 5.